container-chains/nodes/simple/src/command.rs

```diff
@@ -592,3 +499,152 @@ impl CliConfiguration<Self> for RelayChainCli {
         self.base.base.node_name()
     }
 }
+
+fn rpc_provider_mode(cli: Cli, profile_id: u64) -> Result<()> {
```

Comment:
maybe this belongs more in service.rs?

Reply:
No other functions taking `Cli` are in service. In the future we'll likely refactor generic part into a common crate and keep the code in commands.rs minimal.

Reply:
roadmap